### ISSUE TITLE
Waning messages during `brew file update`

### ISSUE THREAD

**Issue by crhg** [tholu/homebrew-tap#2]
The following warning messages is generated while executing `brew file update`.

```terminal
% brew file update                                                                               [~]
Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/tholu/homebrew-tap/subversion18.rb:46:in `<class:Subversion18>'
Please report this to the tholu/tap tap!

Warning: Calling <<-EOS.undent is deprecated!
Use <<~EOS instead.
/usr/local/Homebrew/Library/Taps/tholu/homebrew-tap/subversion18.rb:235:in `caveats'
Please report this to the tholu/tap tap!

Warning: Calling <<-EOS.undent is deprecated!
Use <<~EOS instead.
/usr/local/Homebrew/Library/Taps/tholu/homebrew-tap/subversion18.rb:269:in `caveats'
Please report this to the tholu/tap tap!

$ brew update
Already up-to-date.
$ brew upgrade --fetch-HEAD
Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/tholu/homebrew-tap/subversion18.rb:46:in `<class:Subversion18>'
Please report this to the tholu/tap tap!
$ cd /Users/matsui/.brewfile/crhg_Brewfile
$ git pull
Already up-to-date.
Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/tholu/homebrew-tap/subversion18.rb:46:in `<class:Subversion18>'
Please report this to the tholu/tap tap!

Warning: Calling <<-EOS.undent is deprecated!
Use <<~EOS instead.
/usr/local/Homebrew/Library/Taps/tholu/homebrew-tap/subversion18.rb:235:in `caveats'
Please report this to the tholu/tap tap!

Warning: Calling <<-EOS.undent is deprecated!
Use <<~EOS instead.
/usr/local/Homebrew/Library/Taps/tholu/homebrew-tap/subversion18.rb:269:in `caveats'
Please report this to the tholu/tap tap!


###########################################################
# Initialize /Users/matsui/.brewfile/crhg_Brewfile/Brewfile
###########################################################


###################################################################
# You can edit /Users/matsui/.brewfile/crhg_Brewfile/Brewfile with:
#     $ brew-file edit
###################################################################

$ cd /Users/matsui/.brewfile/crhg_Brewfile
$ git add -A
$ git commit -m "Update the package list"
On branch master
Your branch is up-to-date with 'origin/master'.
nothing to commit, working tree clean
$ git push
Everything up-to-date
```

My environment is:

```terminal
% brew -v
Homebrew 1.5.1
Homebrew/homebrew-core (git revision 4020; last commit 2018-01-23)
brew-file v5.1.0 5/Jan/2018
```

**Comment by tholu** [comment]
Fixed in 2f4ac35878a1a9d65729133306c0f6b4f566f729.

**Comment by crhg** [comment]
I confirmed that the reported warnings are not outputted. Thank you.
